msvc/libmini-common.targets

```diff
@@ -60,6 +60,8 @@
     <ClCompile Include="$(MonoSourceLocation)\mono\mini\debugger-agent-stubs.c" />
     <ClInclude Include="$(MonoSourceLocation)\mono\mini\debugger-engine.h" />
     <ClCompile Include="$(MonoSourceLocation)\mono\mini\debugger-engine.c" />
+    <ClCompile Include="$(MonoSourceLocation)\mono\mini\debugger-state-machine.c" />
+    <ClCompile Include="$(MonoSourceLocation)\mono\mini\debugger-state-machine.h" />
```

Comment:
ClInclude, not ClCompile

Reply:
Oh my. Thank you so much. I think I looked at that file too many times and became blind to it.